ossdbtoolsservice/connection/connection_service.py

```diff
@@ -137,7 +137,7 @@ def connect(self, params: ConnectRequestParams, request_context: RequestContext
                     del self._cancellation_map[cancellation_key]
 
         # If the connection was canceled, close it
-        if cancellation_token.canceled:
```

Comment:
you can keep it canceled. 
If you dont want to expose this variable directly,
Use the @property.

You can see the usage in mysql driver. We have properties there.

Reply:
done